po/fr.po

```diff
@@ -38,65 +38,69 @@ msgstr ""
 msgid "read;fast;speed;reading;"
 msgstr "lire;vite;rapide;lecture;"
 
-#: src/SpedreadReadTab.vala:79 src/SpedreadWindow.vala:313
+#: src/SpedreadReadTab.vala:79 src/SpedreadWindow.vala:337
 msgid "Go to \"Text\" and paste your read!"
 msgstr "Allez dans \"Texte\" et collez votre lecture !"
 
-#: src/SpedreadWindow.vala:267
+#: src/SpedreadWindow.vala:291
 msgid "End reached"
 msgstr "Terminé"
 
 #. TR: If plural is an issue, you can translate it as "Time left: %u min %u s"
-#: src/SpedreadWindow.vala:292
+#: src/SpedreadWindow.vala:316
 #, c-format
 msgid "%u min %u s left"
 msgstr "Temps restant : %u min %u s"
 
-#: src/SpedreadWindow.vala:427
+#: src/SpedreadWindow.vala:451
 msgid "Text"
 msgstr "Texte"
 
-#: src/SpedreadWindow.vala:428
+#: src/SpedreadWindow.vala:452
 msgid "Read"
 msgstr "Lecture"
 
-#: src/SpedreadWindow.vala:463
+#: src/SpedreadWindow.vala:491
 msgid "New Window (Ctrl+N)"
 msgstr "Nouvelle Fenêtre (Ctrl+N)"
 
-#: src/SpedreadWindow.vala:493
+#: src/SpedreadWindow.vala:521
 msgid "Main Menu"
 msgstr "Menu Principal"
 
-#: src/SpedreadWindow.vala:560
+#: src/SpedreadWindow.vala:601
 msgid "This change will only be applied after you restart Spedread"
 msgstr "Ce changement ne sera appliqué qu'après avoir relancé Spedread"
 
-#: src/SpedreadWindow.vala:564
+#: src/SpedreadWindow.vala:605
 msgid "_OK"
 msgstr "_OK"
 
-#: src/SpedreadWindow.vala:585
+#: src/SpedreadWindow.vala:626
 msgid "About Spedread..."
 msgstr "À propos de Spedread..."
 
 #. TR: "Name <email@domain.com>", "Name https://website.example" or "Name"
-#: src/SpedreadWindow.vala:590
+#: src/SpedreadWindow.vala:631
 msgid "translator-credits"
 msgstr "Naqua Darazaki <n.darazaki@gmail.com>"
```

Comment:
Since you've translated a string to french, you should add yourself to the french translation credits here! All work deserves reward (except it's free software so we pay in exposure...)

You can see an example of how it's done here: https://github.com/GNOME/loupe/blob/273dc2f4e77ae991e3589787d6c5e3513d94c672/po/fr.po#L297-L304